Jetchat/app/src/main/java/com/example/compose/jetchat/DroidconEmbeddingsWrapper.kt

```diff
@@ -202,31 +210,43 @@ class DroidconEmbeddingsWrapper(val context: Context?) {
             }
             if (handled) {
                 // add the 'call a function' response to the history
-                conversation.add(
-                    ChatMessage(
-                        role = completionMessage.role,
-                        content = completionMessage.content
-                            ?: "", // required to not be empty in this case
-                        functionCall = completionMessage.functionCall
-                    )
+                val funcCall = ChatMessage(
+                    role = completionMessage.role,
+                    content = completionMessage.content
+                        ?: "", // required to not be empty in this case
+                    functionCall = completionMessage.functionCall
                 )
+                //Log.v("LLM-SW", "-- message ${completionMessage.role.role}: ${completionMessage.functionCall?.arguments}")
+                //Log.v("LLM", "        contains tokens: ${TokenizerHelper.countTokensIn(completionMessage.functionCall?.arguments)}")
```

Comment:
hm i think it would make sense to either uncomment or just delete these logs! same with the ones below

Reply:
good call, these messages are now evaluated inside the SlidingWindow function... I'd put the logging in prior to that 😬